app/models/bus.rb

```diff
@@ -0,0 +1,5 @@
+class Bus < ApplicationRecord
+  validates :bus_name, :bus_type, :bus_brand, :bus_capacity, presence:true
```

Comment:
IMHO, we do not need to repeat the model name for each attribute name. The model is already named as `Bus`, so the attributes are definitely bus-attributes, right? Repeating the model name each time is too wordy, don't you think?

Reply:
Yes. renamed those.